components/org.wso2.carbon.status.dashboard.web/src/error-pages/Error403.jsx

```diff
@@ -22,13 +22,13 @@ import Header from "../common/Header";
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 import darkBaseTheme from 'material-ui/styles/baseThemes/darkBaseTheme';
 import getMuiTheme from 'material-ui/styles/getMuiTheme';
-import {RaisedButton} from "material-ui";
+import {Card, CardText, CardTitle, RaisedButton} from "material-ui";
 import {Link} from "react-router-dom";
+import FormPanel from "../common/FormPanel";
 const muiTheme = getMuiTheme(darkBaseTheme);
-const buttonStyle = {marginLeft: 60, width: 100, fontSize: '12px',backgroundColor:'#f17b31'};
-
+const buttonStyle = {position: 'center', width: '15%', fontSize: '12px',backgroundColor:'#f17b31'};
```

Comment:
Check formatting

Reply:
whitespaces after comma

Reply:
Fixed in 4951a59207b2595a47dcb672ab613964fdceb93b